### DISCUSSION TITLE
User Manual

### DISCUSSION THREAD

**Discussion by User123456789012345678901234567890** [nuttyartist/notes#423] [Ideas]
Is there a basic User Manual that you can read to familiarize yourself with the notes app? If there isn't, please create one if possible (Include formatting, accessibility, etc)! That would be really appreciated!

**Comment by nuttyartist** [comment]
Hi! How about our website? It shows different use cases and things you can do. https://www.get-notes.com
Do you think we should extend it even further? What would you like to see?
